.travis.yml

```diff
@@ -1,19 +1,56 @@
 language: java
 sudo: required
 dist: trusty
+
+env:
+  matrix:
+    - ENGINE=lucee@4.5
+    - ENGINE=lucee@5
+    - ENGINE=adobe@10
+    - ENGINE=adobe@11
+    - ENGINE=adobe@2016
+
+cache:
+  directories:
+  - $HOME/.CommandBox
+
 before_install:
-- sudo apt-key adv --keyserver keys.gnupg.net --recv 6DA70622
-- sudo echo "deb http://downloads.ortussolutions.com/debs/noarch /" | sudo tee -a
-  /etc/apt/sources.list.d/commandbox.list
+  - sudo apt-key adv --keyserver keys.gnupg.net --recv 6DA70622
+  - sudo echo "deb http://downloads.ortussolutions.com/debs/noarch /" | sudo tee -a
+    /etc/apt/sources.list.d/commandbox.list
+
 install:
-- sudo apt-get update && sudo apt-get --assume-yes install commandbox
-- cd tests
-- box install
-- box server start
+  # Install CommandBox and our JSON parsing binary
+  - sudo apt-get update && sudo apt-get --assume-yes install commandbox jq
```

Comment:
I think Travis must have added some additional entropy, because I'm not having any trouble starting ACF servers now. I could certainly add it, though, if it speeds up the build.

`jq` is the JSON parsing library I'm using to pull the version number from the `box.json` file. See implementation below.